iroh-rpc-client/src/status.rs

```diff
@@ -1,81 +1,10 @@
 use crate::{gateway, network, store};
 use anyhow::Result;
-use async_stream::stream;
-use futures::Stream;
-use tonic::transport::channel::Channel;
-use tonic_health::proto::{
-    health_check_response::ServingStatus, health_client::HealthClient, HealthCheckRequest,
-    HealthCheckResponse,
-};
 
 // TODO: make configurable
-const WAIT: std::time::Duration = std::time::Duration::from_millis(1000);
+const _WAIT: std::time::Duration = std::time::Duration::from_millis(1000);
```

Comment:
If this isn't used anymore maybe delete it?

Reply:
I think it is related to the status stuff that was entirely removed.